test/e2e/specs/all-features-confirmation/errors/errors.spec.js

```diff
@@ -0,0 +1,85 @@
+var chaisePage = require('../../../utils/chaise.page.js');
+var recordHelpers = require('../../../utils/record-helpers.js');
+var testParams = {
+    table_name: "accommodation",
+};
+errorTexts = {
+  deletionErrText : "This entry cannot be deleted as it is still referenced from the booking table. All dependent entries must be removed before this item can be deleted.\n\nClick OK to go to the Record Page Show Error Details",
+  uniqueConstraint : "Error The entry cannot be created/updated. Please use a different ID for this record."
+};
+
+/*
+*  All error related test cases are created here.
+*  Test cases from other apps would be collated in future
+*/
+
+describe('Error related to Record App,', function() {
+
+    describe("For no record found in Record app", function() {
+
+        beforeAll(function() {
+          browser.ignoreSynchronization=true;
+            var url = browser.params.url + "/record/#" + browser.params.catalogId + "/product-record:" + testParams.table_name +  "/id=11223312121";
```

Comment:
Also factor out the schema name here into a variable so if the schema changes, we can easily change the tests.